src/ddmd/expression.d

```diff
@@ -8520,6 +8520,7 @@ extern (C++) final class DotIdExp : UnaExp
 {
     Identifier ident;
     bool noderef;       // true if the result of the expression will never be dereferenced
+    bool wantsym;       // do not replace Symbol with its initializer during semantic()
```

Comment:
Yes, it's exact. Normally, the replacement of a symbol with its initializer is supposed to be in semantic2(). But Kenji moved it to semantic() for enums. I did not revert that out of concern for it being a breaking change, hence opted for the simple flag.